workers.go

```diff
@@ -59,19 +59,20 @@ type WorkerRouteResponse struct {
 // WorkerScript Cloudflare Worker struct with metadata.
 type WorkerScript struct {
 	WorkerMetaData
-	Script            string   `json:"script"`
-	UsageModel        string   `json:"usage_model,omitempty"`
-	CompatibiltyFlags []string `json:"compatibility_flags,omitempty"`
-	CompatibiltyDate  string   `json:"compatibility_date,omitempty"`
```

Comment:
Are you sure? Unless I'm missing something they should be equivalent: https://go.dev/play/p/QZ3qotBU9fU

It's hard to tell since the values aren't in the test responses and there's no test asserting on these values in the response that I can see.

That said, they probably make more sense in WorkerMetaData, though I'm not sure why WorkerMetaData is separate to begin with...

Reply:
@davars 
> Are you sure? 
Or at least last time I tested them they were not getting returned. It might as well be that the Cloudflare API has changed internally and the response from the CF server has more data now. Either way, I agree that they should be in the metadata.